contracts/scripts/core/DeployCore.s.sol

```diff
@@ -32,8 +32,8 @@ contract DeployTestnet is Script {
         uint16 feePercent = 2;
         uint16 providerPenaltyPercent = 5;
         uint64 commitmentDispatchWindow = 2000;
-        uint256 withdrawalDelay = 24 * 3600 * 1000; // 24 hours in milliseconds
-        uint256 protocolFeePayoutPeriodBlocks = 5 * 3600; // 1 hour with 200ms blocks
+        uint256 withdrawalDelay = 24  hours  * 1000; // 24 hours in milliseconds
```

Comment:
24 hours right? If so we shouldn't multiply by 1000

Reply:
It's in miliseconds for our chain.